Sources/MongoSwift/MongoCursor.swift

```diff
@@ -3,35 +3,38 @@ import Foundation
 import NIO
 import NIOConcurrencyHelpers
 
-internal let ClosedCursorError = LogicError(message: "Cannot advance a completed or failed cursor.")
+internal struct MongocCursor: MongocCursorWrapper {
```

Comment:
similarly to what you did on the change stream wrapper, probably helpful to note here that this is a wrapper of a `mongoc_cursor_t`